rfd/0089-merge-webapps.md

```diff
@@ -0,0 +1,141 @@
+---
+authors: Jeff Pihach (jeff.pihach@goteleport.com)
+state: draft
+---
+
+# RFD 89 - Merge Webapps into Teleport
+
+## What
+
+- Merge the [webapps](https://github.com/gravitational/webapps) repo into the
+  [teleport](https://github.com/gravitational/teleport) repo.
+- Merge the [webapps.e](https://github.com/gravitational/webapps.e) repo into
+  the [teleport.e](https://github.com/gravitational/teleport.e) repo.
+
+## Why
+
+The version of the web UI and the Teleport are tightly coupled but the
+repositories are separate which means that complexities arrise when integrating
+the two projects. We have to separately clone and build the web UI and commit
+the webasset changes into the teleport repo whenever we want to represent an
+update. We also have to maintain multiple build systems and ops procedures for
+the web UI. Moving the web UI repositories into their respective Teleport repo
+will save us time and confusion.
+
```

Comment:
Might be worth adding that the webapps monorepo was developed at a time when we were supporting the UI for both Gravity and Teleport. Now that Gravity is no more, it makes more sense to combine them.

Reply:
done